robottelo/ui/contentviews.py

```diff
@@ -30,6 +30,21 @@ def go_to_filter_page(self, cv_name, filter_name):
         strategy, value = locators['contentviews.select_filter_name']
         self.click((strategy, value % filter_name))
 
+    def set_calendar_date_value(self, name, value=None):
+        """Set the input value of a date field and press the button to hide
+         the calendar popup panel"""
+        date_element = self.wait_until_element(
+            locators['contentviews.calendar_date_input'] % name)
+        date_element.clear()
+        if value:
+            date_element.send_keys(value)
```

Comment:
this is a set value function, if a user do want to change the value it have to not set it's value (by calling this function), will redo back this signature for value without default value None